values.yaml

```diff
@@ -53,13 +51,23 @@ provision:
     existingConfigmap: ""
     configmapKey: "users.js"
 
+# Additional GraphDB properties to insert in the properties configmap
+properties: {}
```

Comment:
we should have something similar for the proxy.

Reply:
It's on my TODO list :smiley: